bin/stats.py

```diff
@@ -48,8 +49,8 @@ def problem_stats(problems):
         # Roughly in order of importance
         ("  time", lambda p: p.limits.time_limit, 0),
         ("yaml", "problem.yaml"),
-        ("tex", "problem_statement/problem*.tex", 1),
-        ("sol", "problem_statement/solution*.tex", 1),
+        ("tex", str(latex.PdfType.PROBLEM.path(ext="*.tex")), 1),
+        ("sol", str(latex.PdfType.SOLUTION.path(ext="*.tex")), 1),
```

Comment:
For the other globs, you wrote `str(latex.PdfType.PROBLEM.path("*"))`. Do we want that here as well?

Reply:
hm i did this to match the old glob (`*.tex` vs `.*.tex`) but  yes `.*.tex` should be fine here

Reply:
In that case, maybe it _was_ intentional that the glob also matched `problem-slide.*.tex` :joy: But as agreed upon, we don't have to do that anymore here :slightly_smiling_face: